Logger.Tests/FileLoggerTests.cs

```diff
@@ -5,6 +5,21 @@ namespace Logger.Tests
     [TestClass]
     public class FileLoggerTests
     {
-        
+        [TestMethod]
+        public void FileLogger_FilePathIsCreated()
+        {
+            // Arrange
+            LogFactory logFac = new LogFactory();
+
+            // Act
+            //Path for file is subject to change if folder location is moved for the text file.
+            logFac.ConfigureFileLogger("P:/.Net Programming Assignment 2/Logger/test.txt");
+            FileLogger? fileLog = (FileLogger?)logFac.CreateLogger(nameof(FileLoggerTests));
```

Comment:
```suggestion
            FileLogger fileLog = new FileLogger("test.txt");
```
Generally speaking, when unit testing you only want to test one _unit_ of your code at a time. Since you are testing your `FileLogger` and not your `LogFactory` you should directly instantiate a new `FileLogger`.